pallets/registry/Cargo.toml

```diff
@@ -0,0 +1,67 @@
+[package]
```

Comment:
Is this registry code similar to pallet-registry which already there? Or any changes in logic? (other than removing tight coupling with namespace?)